### ISSUE TITLE
[slider] accessibility and no visible value label

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-web-components#1159]
It's currently possible to hide the value labelling via the `getAriaValueText` property as follows:

```
    <sp-slider
        label="Slider Label"
        max=100
        min=0
        .getAriaValueText=${() => ''}
      ></sp-slider>
```

This was originally developer to allow for label customization, a la:

```
    <sp-slider
        label="Slider Label"
        max=100
        min=0
        .getAriaValueText=${(value) => `${value}px`}
      ></sp-slider>
```

This customization is meant to allow for more correct accessible content in the element. When leveraging it to _remove_ the value label, is that bad for a11y and should we do something different (e.g. hide it visible, or similar) that is maybe triggered via a more specific attribute/boolean based API?

**Comment by adixon-adobe** [comment]
I'd like to see a Boolean API to control this. Something like `hide-value-label`. We just enabled a case where we want this (a thumbnail size slider where the visual label would add confusion). We were able to support this with some sneaky css that will likely need maintaining as new slider versions are integrated:

```
sp-slider {
        --spectrum-fieldlabel-m-padding-top: 0;
	    --spectrum-label-text-size: 0;
	    --spectrum-label-text-line-height: 0;
}
```

**Comment by Westbrook** [comment]
@adixon-adobe in this context are you also removing the text label visible to left as well?

**Comment by adixon-adobe** [comment]
Ahhh -- yes, we are, so it's not exactly the same.

**Comment by Westbrook** [comment]
A solution that can support the multiplicity here `text label only`, `value label only`, `mix`, `neither` would of course be optimal...

**Comment by cuberoot** [comment]
Would two attributes work? e.g. `hide-label` and `hide-value-label`?

**Comment by Westbrook** [comment]
This is tricky because while nothing is stopping you from leveraging `aria-label` in this case, you won't actually be labelling the input element(s) as required for the pattern to be accessible in this case. I think you need some level of complex attribute to reference both the "text label" and the "value label".

In this way maybe, `visible-label="text|value|none"`? With the default (or lack of the `visible-label` attribute) outlining "both", as is the case now by default.
